kiez/kiez.py

```diff
@@ -160,7 +167,7 @@ def kneighbors(
         source_query_points=None,
         k=None,
         return_distance=True,
-    ) -> Union[np.ndarray, Tuple(np.ndarray, np.ndarray)]:
+    ) -> Union[np.ndarray, Tuple[np.ndarray, np.ndarray]]:
```

Comment:
this isn't related to the PR, but it needed to be fixed to pass CI

Reply:
I think it is because you deleted the imports for Tuple and Any before

Reply:
type annotations require square bracket syntax and not round brackets